### ISSUE TITLE
Crash when switching to Alsa default device

### ISSUE THREAD

**Issue by Chaircrusher** [VCVRack/Rack#1501]
### Details
Repeatable crash if I select the 'default (1-1 in, 1-2 out)' ALSA device 
Operating system: Centos 7 (Linux gozgo 3.10.0-957.21.3.el7.x86_64)
Rack version: v1.1.3
Hardware relevant to your issue (e.g. graphic card model, audio/MIDI device):
I'm using a USB DAC module that Linux identifies as X-DA2.

### Summary
If I select the 'default' ALSA device, Rack crashes, down in the ASound directory somewhere.

### Steps to reproduce
1. Load any patch with an AUDIO-8 module.
2. Select the last option for audio device which is "default (1-1 in, 1-2 out)
[log.txt](https://github.com/VCVRack/Rack/files/3428250/log.txt)

**Comment by Chaircrusher** [comment]
Also v1.1.4

**Comment by lunakid** [comment]
Just arrived with a related crash (stall, actually): unselecting ASIO(! not ALSA) device, back to "(No device)":

![image](https://user-images.githubusercontent.com/698307/70263799-75148100-1797-11ea-98a3-7f3b2eb5bdaa.png)

**Comment by Petervos2018** [comment]
Don't use 2 audio devices ! use only the audio-16 or the audio-8 or wait for Rack V2 ;)

**Comment by lunakid** [comment]
> Don't use 2 audio devices ! use only the audio-16 or the audio-8 or wait for Rack V2 ;)

Thanks, but I never intended to use 2. I wanted to replace the AUDIO-8 with 16, but obviously wanted to see first if the new one works at all (these are my first steps with VCV), before deleting the old (as per the usual way of replacing thing X with thing Y avoiding a potential dead-end).

Notice also, how there's no audio device selected in the (newly added) 16-bit module. 
And I just wanted to "unuse" the audio device in the AUDIO-8 first, so then I wuld supposedly have not 2, but 0 audio devices used.

**Comment by Petervos2018** [comment]
>Notice also, how there's no audio device selected in the (newly added) 16-bit module.

That was the moment that VCV rack crashed ;)
Just remember (for now) only 1 audio module in a patch.
Have fun.

**Comment by lunakid** [comment]
> > Notice also, how there's no audio device selected
> 
> That was the moment that VCV rack crashed ;) 

Which, umm, it shouldn't, I guess? ;) 
Also: "no audio device being selected" alone apparently did not cause any problem e.g. when the demo patch was initially loaded by default. There was just no sound. As expected. So, it also shouldn't crash with a new audio module loaded with no device selected. (Well, not ever, actually. ;) )

> Oh and it's not 8bit 16bit its 8 and 16 channels to and from your sound card.

Yes, I realized that (in fact that's why I came back here: to edit that mistake), as soon as it finally recovered from the crash: because the next run was dead in the water, already stuck unresponsive right after launch, as soon as it loaded the same config.
But luckily it recovered on the next launch.

> Have fun.

Doin' it, thanks! :)

> Also look here : https://vcvrack.com/manual/Core.html

I did, when I tried to figure out why there was no sound, no matter what I did.
(That's another story, why it did nothing when I desperately clicked on the grayed strip of the audio device. I gave up, and retried the next day, and then it suddenly responded, but only after a few futile attempts again. (No other audio app was open holding the device in either case.))

Don't get me wrong, still a great toy already! :+1:

**Comment by AndrewBelt** [comment]
@lunakid Your issue is different than OP's. OP's cause is likely with the ALSA driver. Your cause is with opening multiple VCV Audio modules. Your issue is a duplicate of https://github.com/VCVRack/Rack/issues/132.

**Comment by lunakid** [comment]
> @lunakid Your issue is a duplicate of #132.

Thank you, Andy, for the link. (Avoiding a duplicate was the reason I haven't opened a new one, and ended up here by picking the first close match from the list of `"is:issue is:open crash"` (assuming it unlikely to have two unrelated crashes around changing the audio device); and #132 was not there. (As it's closed, I've read it now.) Thanks again guys, @Petervos2018, too!)
